.enterprise/scripts/is-at-master.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
isn't it simpler to get the latest commit in master and then compare it against the `.enterprise-commit` ?

```
git ls-remote $ENTERPRISE_REPO | grep refs/heads/master | cut -f 1
```

Reply:
this can be done as well, I am not sure how to get a nice output in this case, however.

Reply:
I will actually make it this way since it is faster and simpler. I can sacrifice the nice error output.

Reply:
so if I am working on a feature branch with changes in the enterprise repo, this check will fail until the very last moment?
I only wish we were able to do fast-forward merges on GitHub's pull requests...! :)

Reply:
Ok, I made the changes. Yes, it will fail until the last moment the way we currently do it.

It is a separate action and it takes 5sec to run so it shouldn't create any more trouble other than the ❌ 


We can discuss ff merges but given the situation I think we need this

Reply:
it is not possible to have ff merges in GitHub according to the docs.

Reply:
good point, will check

Reply:
We are using [pull_request](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) as a trigger, it is using a merged ref to run the actions on.